data/changelog/dune/2019-02-12-dune-1.7.0.md

```diff
@@ -151,7 +151,7 @@ On behalf of the dune team, I'm pleased to announce the release of dune 1.7.0. T
 
 * The virtual libraries feature has made it to 1.7.0. I've blogged about it [before](http://rgrinberg.com/posts/virtual-libraries/)
 
-* There's now a [mode](https://dune.readthedocs.io/en/latest/advanced-topics.html#implicit-transitive-deps) to disable transitive dependencies. This mode still has some caveats because it lacks proper compiler support. But we welcome user reports regarding its usability.
+* There's now a [mode](https://dune.readthedocs.io/en/latest/advanced-topics.html#implicit-transitive-deps - [404 Not Found]) to disable transitive dependencies. This mode still has some caveats because it lacks proper compiler support. But we welcome user reports regarding its usability.
```

Comment:
```suggestion
* There's now a [mode](https://dune.readthedocs.io/en/stable/dune-files.html) to disable transitive dependencies. This mode still has some caveats because it lacks proper compiler support. But we welcome user reports regarding its usability.
```
This link was the closest I could find